src/views/ProjectView.vue

```diff
@@ -63,11 +63,12 @@ const filters = ref({
   },
   submitter: {
     value: route.query.submitter || '',
-    label: 'Submitter',
+    label: 'Submitter User ID',
   },
 })
 let filtersDebounceTimer
 const FILTER_DEBOUNCE = 600
+const DEFAULT_DATE_RANGE_DAYS = -30
```

Comment:
Seems strange having a negative value for a constant that says its days. Maybe just make this positive but put a negative sign in front of it when you want to use it to subtract days from something?